server/services/interventionsService.ts

```diff
@@ -62,6 +63,11 @@ export interface UpdateDraftEndOfServiceReportParams {
   outcome: CreateEndOfServiceReportOutcome | null
 }
 
+export interface CreateReportDateParams {
+  fromDate: string | null
```

Comment:
what happens if either of these is null?

Reply:
The `null` behaviour is taken from my other branch that uses this function, but I think I might refactor it so it's not necessary. Have updated this to remove the `null`s.